app/server/src/client/index.ts

```diff
@@ -5,7 +5,6 @@ export {
   addParameters,
   configure,
   getStorybook,
-  forceReRender,
```

Comment:
I'm guessing this was an old field - not present any more once type-safety was added.